### ISSUE TITLE
Crash from Android.AppCompat.NavigationPageRenderer

### ISSUE THREAD

**Issue by kensykora** [xamarin/Xamarin.Forms#5556]
### Description

From crash reports we collect in MS AppCenter, we identified a crash that occurred (twice) from an Android NavigationPageRenderer. We unfortunately weren't able to reproduce it internally but wanted to post it here to track and see if it is a known issue (we're running Xamarin.Forms 3.2) that's already been addressed potentially.

```
Xamarin Exception Stack:
System.NullReferenceException: Object reference not set to an instance of an object
  at Xamarin.Forms.Platform.Android.AppCompat.NavigationPageRenderer+<UpdateBitmap>d__89.MoveNext () [0x0013c] in <b9bd728dc3884553a65889a49f5d7aad>:0
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state) [0x00000] in <c66da030ede340e0a105f7beb2c58fe4>:0
  at Android.App.SyncContext+<>c__DisplayClass2_0.<Post>b__0 () [0x00000] in <de4f2a2c9b4a447b8e5a6413c245c751>:0
  at Java.Lang.Thread+RunnableImplementor.Run () [0x00008] in <de4f2a2c9b4a447b8e5a6413c245c751>:0
  at Java.Lang.IRunnableInvoker.n_Run (System.IntPtr jnienv, System.IntPtr native__this) [0x00009] in <de4f2a2c9b4a447b8e5a6413c245c751>:0
  at (wrapper dynamic-method) System.Object.40(intptr,intptr)
```

See attached full crash report here: [crash-report.txt](https://github.com/xamarin/Xamarin.Forms/files/2966347/crash-report.txt)

### Basic Information

- Version with issue: 3.2.0.871581
- Last known good version: unknown
- Platform Target Frameworks: 
  - Android: 8.1
- Android Support Library Version: 28.0.0.1
- Nuget Packages: n/a
- Affected Devices: LG Tribute Dynasty (Android 7.1.2)

**Comment by samhouts** [comment]
It's very possible that this was fixed in 3.3 with #3375. I can't prove that without a reproduction, however. I recommend trying to update your project and see if that prevents the crashes. Otherwise, I'll put this into the Needs Info category in case you can find a way to reproduce it. Thanks!

**Comment by samhouts** [comment]
@kensykora  Since we haven't heard from you in more than 30 days, we hope this issue is no longer affecting you. If it is, please reopen this issue and provide the requested information so that we can look into it further. Thank you!
